src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
```

Comment:
[CODE KNOWLEDGE] - Similar to the `add` method, `subtract` should only take one parameter and subtract it from the current `result`.